src/winter_onboarding_2021/fleet_management_service/handlers/core.clj

```diff
@@ -9,3 +9,7 @@
 (defn health-check [_]
   (response/response {:message "Everything OK!"
                       :data (healthcheck/healthcheck-query)}))
+
+(defn root [_]
+  {:title "Not Found"
+   :content ""})
```

Comment:
Why did we name this `root`?  Won't `not-found` be better?

Reply:
Will add 'Not found' to content